### PR TITLE
Return timestamp strings as ISO format

### LINKED ISSUE #189: Remove timestamp from the API models and replace with String type

### PR DESCRIPTION
**Pull request by wslulciuc**
This PR introduces the following changes:

1. Sets server timezone to UTC using the JVM argument `user.timezone`
2. Drops timezones from timestamps stored in DB
3. Formats all API timesamps as ISO format
4. ~Renames `db_table_infos` to `db_table_info`~

Closes #189 #183

### CODE REVIEW

File: src/test/java/marquez/db/mappers/DbTableVersionRowMapperTest.java [hunk 1]
```diff
@@ -40,7 +40,10 @@
 
   @Test
   public void testMap() throws SQLException {
+    final Object exists = mock(Object.class);
     final ResultSet results = mock(ResultSet.class);
+    when(results.getObject(Columns.CREATED_AT)).thenReturn(exists);
```
Comment (ashulmanWeWork) [inline src/test/java/marquez/db/mappers/DbTableVersionRowMapperTest.java]: How does this reconcile with line 48?
```
when(results.getTimestamp(Columns.CREATED_AT)).thenReturn(Timestamp.from(CREATED_AT));
```

Comment (wslulciuc) [reply src/test/java/marquez/db/mappers/DbTableVersionRowMapperTest.java]: Sure, see [`Columns.timestampOrThrow()`](https://github.com/MarquezProject/marquez/blob/master/src/main/java/marquez/db/Columns.java#L93)

File: src/main/resources/db/migration/V19__add_created_at_column_to_db_info_table.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE db_table_infos RENAME TO db_table_info;
```
Comment (sshah-wework) [inline src/main/resources/db/migration/V19__add_created_at_column_to_db_info_table.sql]: this seems unrelated -- does the `DbTableInfoDao` have to change now? Also seems like we don't have `DbTableInfoDao` tests to catch this. Filed #485 

The DAO method below might need to be updated?
```
  @SqlUpdate(
      "INSERT INTO db_table_infos (uuid, db_name, db_schema_name) VALUES (:uuid, :db, :dbSchema)")
  void insert(@BindBean DbTableInfoRow dbTableInfoRow);
```

File: src/main/resources/db/migration/V20__add_update_at_column_to_job_runs_table.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+ALTER TABLE job_runs ADD COLUMN updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP;
```
Comment (sshah-wework) [inline src/main/resources/db/migration/V20__add_update_at_column_to_job_runs_table.sql]: needs to be `NOT NULL`able?

File: src/main/resources/db/migration/V19__add_created_at_column_to_db_info_table.sql (hunk 2 above)
Comment (wslulciuc) [reply src/main/resources/db/migration/V19__add_created_at_column_to_db_info_table.sql]: Sure, let's update the table name when we fix #485. Thanks for filling the issue, @sshah-wework!

File: src/main/resources/db/migration/V20__add_update_at_column_to_job_runs_table.sql (hunk 3 above)
Comment (wslulciuc) [reply src/main/resources/db/migration/V20__add_update_at_column_to_job_runs_table.sql]: Yes!
